pkg/lang/ir/compile.go

```diff
@@ -114,7 +112,7 @@ func CompileEntrypoint(buildContextDir string) ([]string, error) {
 }
 
 func CompileEnviron() []string {
-	if DefaultGraph.Image != nil {
+	if !DefaultGraph.DevTools {
```

Comment:
Is there a better name for `dev_tools()`?

Reply:
Any suggestions?

Reply:
- enable_development()
- dev_env()

BTW, should we enable dev tools by default? Or disable it by default? Here are two different designs.

```
def build():
    dev_tools()
```

```
def serve():
    disable_dev()

Reply:
Usually, a function is used to enable some features.

enable_development() is too long to type. dev_env() looks okay, but it actually doesn't add produce an env. Instead, it adds some tools and configurations.